package.json

```diff
@@ -1,38 +1,39 @@
 {
-	"name": "live-server",
-	"version": "0.7.1",
-	"description": "simple development http server with live reload capability",
-	"keywords": [
-		"front-end",
-		"development",
-		"tool",
-		"server",
-		"http",
-		"cli"
-	],
-	"author": "Tapio Vierros",
-	"dependencies": {
-		"connect": "2.x.x",
-		"send": "latest",
-		"colors": "latest",
-		"open": "latest",
-		"watchr": "2.3.x",
-		"faye-websocket": "0.9.x",
-		"event-stream" : "latest"
-	},
-	"scripts": {
-		"test": "jshint *.js"
-	},
-	"bin": {
-		"live-server": "./live-server.js"
-	},
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/tapio/live-server.git"
-	},
-	"engines": {
-		"node": ">=0.8.0"
-	},
-	"preferGlobal": true,
-	"license": "MIT"
+  "name": "live-server",
+  "version": "0.7.2",
+  "description": "simple development http server with live reload capability",
+  "keywords": [
+    "front-end",
+    "development",
+    "tool",
+    "server",
+    "http",
+    "cli"
+  ],
+  "author": "Tapio Vierros",
+  "dependencies": {
+    "connect": "2.x.x",
+    "send": "latest",
+    "colors": "latest",
+    "open": "latest",
+    "watchr": "2.3.x",
+    "faye-websocket": "0.9.x",
+    "event-stream": "latest",
+    "http-proxy": "^1.11.1"
+  },
+  "scripts": {
+    "test": "jshint *.js"
+  },
+  "bin": {
+    "live-server": "./live-server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tapio/live-server.git"
+  },
+  "engines": {
+    "node": ">=0.8.0"
+  },
+  "preferGlobal": true,
+  "license": "MIT"
 }
```

Comment:
Just a note, months late, when you run an `npm` command, it often rewrites the `package.json` file itself and it indents with two spaces.  Likely this re-indentation was a result of npm rewriting the file.